sdcm/sct_config.py

```diff
@@ -1128,7 +1128,29 @@ class SCTConfiguration(dict):
              help="Limit severity level for event types"),
 
         dict(name="scylla_rsyslog_setup", env="SCT_SCYLLA_RSYSLOG_SETUP", type=boolean,
-             help="Configure rsyslog on Scylla nodes to send logs to monitoring nodes")
+             help="Configure rsyslog on Scylla nodes to send logs to monitoring nodes"),
+
+        dict(name="data_device", env="SCT_DATA_DEVICE", type=str,
+             help="""Define which device to use for scylla data. Default for gce is attached. Default for aws is instance_store.
+                     attached - means use additionaly attached devices
+                     instance_store - means use emphereal local NVMes devices(AWS)""",
+             choices=("attached", "instance_store")),
+
+        dict(name="aws_ebs_volume_num", env="SCT_AWS_EBS_VOLUME_NUM",
```

Comment:
@bentsi , what if we generalize it so that it could be reused to gce and azure ?

`aws_ebs_volume_num` -> `additional_volume_num`
`aws_ebs_volume_type` -> `additional_volume_type`
`aws_ebs_volume_size` -> `additional_volume_size`
`aws_ebs_volume_iops` -> `additional_volume_iops`

Reply:
agree

Reply:
fixed